### PR TITLE
Run prettier on docs snippets

### PR DESCRIPTION
**Pull request by kasperpeulen**
## What I did

I thought it might be nice to run prettier on all our snippets, and check it in the CLI.

### CODE REVIEW

File: docs/snippets/angular/badge-story-starter-example.mdx.mdx
Comment (ndelangen) [inline docs/snippets/angular/badge-story-starter-example.mdx.mdx:2]: Not related to this PR, @shilman this HTML-comment style doesn't work with MDX2 anymore, does it?

File: docs/package.json
Comment (ndelangen) [inline docs/package.json:12]: We should not introduce a package-json

File: .circleci/config.yml
Comment (ndelangen) [inline .circleci/config.yml:216]: Let's use the scripts directory for this purpose!
Let's not add another place where we install node-modules.

We could add a npm-script to the root package.json, or a npm-script to the code dir (where we have a bunch of npm-scripts related to linting already)

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: .circleci/config.yml
Comment (kasperpeulen) [reply .circleci/config.yml:216]: Sounds good to me!

**Review by kylegach — COMMENTED** [review]
Maybe MDX 2 makes this less of an (or an entirely moot) issue, but you have be careful running formatting commands on MDX, because it can break the whitespace requirements.

If that's still the case, we should only implement this once our snippets are backed functionally by CI.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@kasperpeulen, as MDX stories are being phased out of the documentation and per @kylegach 's comment, we could apply this to focus only on JS and TS examples to avoid having you blocked on getting it merged as it is a great addition to only not us maintainers but everyone willing to contribute to the docs. With this in mind, if you're ok with reverting back the MDX examples to how they were previously, I'd appreciate it.

File: docs/snippets/angular/badge-story-starter-example.mdx.mdx
Comment (jonniebigodes) [reply docs/snippets/angular/badge-story-starter-example.mdx.mdx:2]: @ndelangen, you're absolutely right about that. It actually doesn't. If a user grabs the code snippet here and pastes it in an MDX 2 example, he's treated with the following:
Storybook-comment-error-mdx2

Thanks for the callout nonetheless, really appreciate it 🙏 ! As once the snippets and overall docs are going to be updated for MDX 2, I'll need to keep this in mind and adjust the comment to avoid issues.

File: docs/package.json
Comment (kasperpeulen) [reply docs/package.json]: Changed this

File: .circleci/config.yml
Comment (kasperpeulen) [reply .circleci/config.yml]: And this

**Review by ndelangen — APPROVED** [review]

**Review by jonniebigodes — APPROVED** [review]
Looks great to me @kasperpeulen ! Thanks for addressing the feedback so promptly! Appreciate it.

**Pull request merged**
